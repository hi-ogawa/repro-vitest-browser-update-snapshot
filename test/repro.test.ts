import { expect, test } from 'vitest';

test('repro', () => {
  expect(2).toMatchSnapshot();
});
