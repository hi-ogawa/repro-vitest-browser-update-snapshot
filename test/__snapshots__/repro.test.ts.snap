// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`repro 1`] = `1`;
